app/main.go

```diff
@@ -62,8 +68,24 @@ func getIPAddress(r *http.Request) string {
 }
 
 func handlerPing(w http.ResponseWriter, r *http.Request) {
-	w.Write([]byte("pong"))
-	fmt.Println(time.Now(), getIPAddress(r), r.Method, r.RequestURI, r.UserAgent())
+	
+	requestBody, err := ioutil.ReadAll(r.Body)
+	n := rand.Intn(100)
+
+	<- time.After(time.Duration(lat) * time.Millisecond)
+	if n <= prob {
+		w.Write([]byte("pong"))
+		fmt.Println(time.Now(), getIPAddress(r), r.Method, r.RequestURI, r.UserAgent())
+	} else {
+		if err != nil {
+			w.WriteHeader(30)
```

Comment:
What's 30 status code =D I did not know this works. https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Reply:
Did I do it wrong?

Reply:
Upps that was a complete accident